### PR TITLE
[ADD] opt login tab

### PR DESCRIPTION
**Pull request by YisusJe**

### CODE REVIEW

File: src/themes/five/src/components/LoginForm/index.js [hunk 1]
```diff
@@ -429,7 +488,7 @@ const LoginFormUI = (props) => {
                       numInputs={numOtpInputs}
                       containerStyle='otp-container'
                       inputStyle='otp-input'
-                      placeholder='0000'
+                      placeholder='000000'
```
Comment (carlosq-ordering) [inline src/themes/five/src/components/LoginForm/index.js:491]: Use theme color

**Review by carlosq-ordering — CHANGES_REQUESTED** [review]

File: src/themes/five/src/components/LoginForm/index.js (hunk 1 above)
Comment (YisusJe) [reply src/themes/five/src/components/LoginForm/index.js:489]: that is only a placeholder

Comment (carlosq-ordering) [reply src/themes/five/src/components/LoginForm/index.js:489]: Use theme color

**Review by carlosq-ordering — CHANGES_REQUESTED** [review]

File: src/themes/five/src/components/LoginForm/index.js [hunk 2]
```diff
@@ -87,6 +87,7 @@ const LoginFormUI = (props) => {
     useLoginOtpCellphone
   } = props
   const numOtpInputs = loginTab === 'otp' ? 6 : 4
+  const otpPlaceholder = [...Array(numOtpInputs)].map((b, i) => i - i).join('')
```
Comment (carlosq-ordering) [inline src/themes/five/src/components/LoginForm/index.js:90]: use `fill` JS method

**Review by carlosq-ordering — CHANGES_REQUESTED** [review]

**Review by carlosq-ordering — APPROVED** [review]

**Pull request merged**